webapp/package.json

```diff
@@ -51,7 +51,8 @@
 		"react-redux": "^7.2.4",
 		"react-router-dom": "^5.2.0",
 		"react-select": "^4.3.0",
-		"react-simplemde-editor": "^5.0.1"
+		"react-simplemde-editor": "^5.0.1",
+		"to-data-view": "^2.0.0"
```

Comment:
Is this change related to the PR?

Reply:
I reverted package.json.  It is no longer changed in the PR.